tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+keeping this __init__.py very light for now. We can expand later if needed. - QHM
```

Comment:
You can leave out initials, the commit log and git blame know who you are! They tend to get left in past the point of usefulness.